package-lock.json

```diff
@@ -21,7 +21,7 @@
       "devDependencies": {
```

Comment:
I think this file should be reverted and the lock file changes done in a separate PR or a publish.